common/transport/mqtt/src/main/java/org/thingsboard/server/transport/mqtt/MqttTransportHandler.java

```diff
@@ -313,6 +313,28 @@ private void processProvisionSessionMsg(ChannelHandlerContext ctx, MqttMessage m
                     sendResponseForAdaptorErrorOrCloseContext(ctx, topicName, msgId);
                 }
                 break;
+            case SUBSCRIBE:
+                MqttSubscribeMessage mqttSubscribeMsg = (MqttSubscribeMessage) msg;
```

Comment:
This implementation is a copy-paste of the original processSubscribe method with minor changes. Please refactor the original method to reuse here. Otherwise this is not acceptable!